rust_dev_preview/tools/toggle_sdk_branch.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
```

Comment:
This script deserves a README or comment explaining how to install dependencies and use it for those that don't regularly use Python.

Reply:
The Code Examples team has agreed on Python as our tooling language, and we therefor expect engineers working on tooling to have an understanding of python. But that isn't publicly documented, so I'll open an issue to write that up for external contributors.